internal/core/metadata/application/notify.go

```diff
@@ -74,7 +74,13 @@ func validateDeviceCallback(ctx context.Context, dic *di.Container, device dtos.
 	request := requests.NewAddDeviceRequest(device)
 	_, err = deviceServiceCallbackClient.ValidateDeviceCallback(ctx, request)
 	if err != nil {
-		return errors.NewCommonEdgeX(errors.KindServerError, "device validation failed", err)
+		// TODO: reconsider the validity in v3
+		// allow this case for the backward-compatability in v2
+		if err.Code() == http.StatusServiceUnavailable {
+			lc.Infof("Skipping device validation for device %s (%s unavailable)", device.Name, device.ServiceName)
```

Comment:
use Warnf

Reply:
"Skipping device validation for device %s due to **device service** %s unavailable"